src/browser.js

```diff
@@ -6,50 +6,63 @@
 
 /* eslint-env browser */
 
-import {FetchToken, createToken} from 'fusion-tokens';
-import {withDependencies} from 'fusion-core';
+import {FetchToken, createOptionalToken} from 'fusion-tokens';
+import {createPlugin} from 'fusion-core';
 import {RPCHandlersToken} from './tokens';
 
-export const RPCRoutePrefixConfigToken = createToken(
-  'RPCRoutePrefixConfigToken'
+export const RPCRoutePrefixConfigToken = createOptionalToken(
```

Comment:
I'm not totally sold on this, but we can go with it for now. I think long term we should look into composition (see https://github.com/fusionjs/fusion-core/issues/90)

Reply:
Agreed this isn't ideal long-term.  I could see us going either the composition route, and would have to provide extensions for all plugins that require `routePrefix` _or_ we move this token into `fusion-tokens` as many plugins will likely come to rely upon this config string.